tests/integration/build_invoke/dotnet/test_build_invoke_dotnet10.py

```diff
@@ -0,0 +1,93 @@
+from tests.integration.build_invoke.build_invoke_base import BuildInvokeBase
+
+"""
+sam build does not support to build dotnetcore 3.1 templates using container, 
+here we put them in a separate file and use a dedicate codebuild project with .net 3.1 runtime to build them.
```

Comment:
Is this comment relevant still? It looks outdated

Reply:
Yeah, removed.